documentation/service-rules/ecf1/transfers.md

```diff
@@ -0,0 +1,232 @@
+- `status` - if leaving end date or joining start date is in the past it is set to `incomplete`, otherwise `complete`
+- `created_at` - set to `leaving_induction_record.created_at`
+- `school_urn` - set to `induction_record.induction_programme.school_cohort.school.urn`
+- `provider` - set to `induction_record.induction_programme.partnership.lead_provider.name`
+- `date` - set to `end_date` for leaving and `start_date` for joining induction record.
+
+## Types of Transfers
+
+### `new_school`
+
+Participant changes to a new school but stays with the same lead provider
+
+```json
+{
+  "data": {
+    "id": "db3a7848-7308-4879-942a-c4a70ced400a",
+    "type": "participant-transfer",
+    "attributes": {
+      "updated_at": "2021-05-31T02:22:32.000Z",
+      "transfers": [
+        {
+          "training_record_id": "000a97ff-d2a9-4779-a397-9bfd9063072e",
+          "transfer_type": "new_school",
+          "status": "complete",
+          "leaving": {
+            "school_urn": "123456",
+            "provider": "Example Institute",
+            "date": "2023-09-01"
+          },
+          "joining": {
+            "school_urn": "654321",
+            "provider": "Example Institute",
+            "date": "2023-09-01"
+          },
+          "created_at": "2021-05-31T02:22:32.000Z"
+        }
+      ]
+    }
+  }
+}
+```
+
+### `new_provider`
+
+Participant changes to a new school with a different lead provider
+
+```json
+{
+  "data": {
+    "id": "db3a7848-7308-4879-942a-c4a70ced400a",
+    "type": "participant-transfer",
+    "attributes": {
+      "updated_at": "2021-05-31T02:22:32.000Z",
+      "transfers": [
+        {
+          "training_record_id": "000a97ff-d2a9-4779-a397-9bfd9063072e",
+          "transfer_type": "new_provider",
+          "status": "complete",
+          "leaving": {
+            "school_urn": "123456",
+            "provider": "Old Institute",
+            "date": "2023-09-01"
+          },
+          "joining": {
+            "school_urn": "654321",
+            "provider": "New Institute",
+            "date": "2023-09-01"
+          },
+          "created_at": "2021-05-31T02:22:32.000Z"
+        }
+      ]
+    }
+  }
+}
+```
+
+### `unknown`
+
+Participant is leaving.
+
+```json
+{
+  "data": {
+    "id": "db3a7848-7308-4879-942a-c4a70ced400a",
+    "type": "participant-transfer",
+    "attributes": {
+      "updated_at": "2021-05-31T02:22:32.000Z",
+      "transfers": [
+        {
+          "training_record_id": "000a97ff-d2a9-4779-a397-9bfd9063072e",
+          "transfer_type": "unknown",
+          "status": "complete",
+          "leaving": {
+            "school_urn": "123456",
+            "provider": "Example Institute",
+            "date": "2023-09-01"
+          },
+          "joining": null,
+          "created_at": "2021-05-31T02:22:32.000Z"
+        }
+      ]
+    }
+  }
+}
+```
+
+## Transfer States
```

Comment:
maybe we can include this table mapping as well for clarity? 
<img width="1058" alt="Screenshot 2025-05-19 at 14 37 34" src="https://github.com/user-attachments/assets/b33395ea-55ad-41cf-9894-dea0c1bb34fa" />